### PR TITLE
Created ColleagueReducer

### PR DESCRIPTION
**Pull request by P-1-X-3-L**

### CODE REVIEW

File: src/Redux/AddColleagueReducer/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { SET_COLLEAGUE_DATA } from '../Types';
+import initialState from './InitialState';
+
+const ColleagueReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_COLLEAGUE_DATA: {
+      return { ...state, data: state.data.concat(action.payload) };
+    }
```
Comment (pranavakkian) [inline src/Redux/AddColleagueReducer/index.js]:
```suggestion
    case SET_COLLEAGUE_DATA: 
      return { ...state, data: state.data.concat(action.payload) };
```

File: src/Redux/AddColleagueReducer/ColleagueReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    expect(ColleagueReducer(undefined, {
+      type: SET_COLLEAGUE_DATA,
+      payload: [{
+        name: 'xyz',
+        jobTitle: 'Engineer',
+        contact: '987654321',
+      }],
+    }))
+      .toStrictEqual({
+        ...initialState,
```
Comment (pranavakkian) [inline src/Redux/AddColleagueReducer/ColleagueReducer.test.js]:
```suggestion
```

File: src/Redux/AddColleagueReducer/ColleagueReducer.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          jobTitle: 'Engineer',
+          contact: '987654321',
+        },
+        {
+          name: 'ABC',
+          jobTitle: 'Cloud Engineer',
+          contact: '123456789',
+        }],
+      });
+  });
```
Comment (pranavakkian) [inline src/Redux/AddColleagueReducer/ColleagueReducer.test.js]: remove this test case. I asked you to do it in single test case, adding one by one.

**Review by pranavakkian — CHANGES_REQUESTED** [review]

File: src/Redux/AddColleagueReducer/ColleagueReducer.test.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+describe('Colleague Data Reducer', () => {
+  test('Setting 1 Colleague Data', () => {
+    expect(ColleagueReducer(undefined, {
+      type: SET_COLLEAGUE_DATA,
+      payload: [{
+        name: 'xyz',
+        jobTitle: 'Engineer',
+        contact: '987654321',
+      }],
+    }))
```
Comment (pranavakkian) [inline src/Redux/AddColleagueReducer/ColleagueReducer.test.js]:
```suggestion
      payload: {
        name: 'xyz',
        jobTitle: 'Engineer',
        contact: '987654321',
      },
    }))
```

**Review by pranavakkian — APPROVED** [review]

**Review by shivalibaghel7 — APPROVED** [review]

**Pull request merged**